### PR TITLE
ADD: Detector time sync functionality

### PR DESCRIPTION
**Pull request by keiranbarr**
Add capability to measure network latency between detector and controller for syncing the timestamps between pose solver and detector. Lays framework to address #81.

NOTE: this work does not actually _account_ for the delay within the pose solver, it just adds the functionality to _measure_ the latency. Future work should focus on accounting for this latency within the pose solver.

### CODE REVIEW

**Comment by tavaughan** [comment]
Related to #85

File: src/common/api/time_sync_stop_request.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from .mct_request import MCTRequest
+from pydantic import Field
+
+
+class TimeSyncStopRequest(MCTRequest):
+    @staticmethod
+    def parsable_type_identifier() -> str:
+        return "time_sync_start_request"
```
Comment (vaughantnrc) [inline src/common/api/time_sync_stop_request.py:8]: Copy-paste error

File: src/common/api/timestamp_get_response.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+
+class TimestampGetResponse(MCTResponse):
+
+    @staticmethod
+    def parsable_type_identifier() -> str:
+        return "timestamp_get_response"
+
+    def timestamp_utc(self):
+        return datetime.datetime.fromisoformat(self.timestamp_utc_iso8601)
```
Comment (vaughantnrc) [inline src/common/api/timestamp_get_response.py:14]: Unused function referencing variable that does not exist

**Review by vaughantnrc — COMMENTED** [review]
I'm closing this PR since I cannot push fixes to it. I'll replace it with another one.

**Pull request merged**